homeassistant/components/livisi/coordinator.py

```diff
@@ -92,6 +93,29 @@ async def async_get_pss_state(self, capability: str) -> bool | None:
         on_state = response["onState"]
         return on_state["value"]
 
+    async def async_get_vrcc_target_temperature(self, capability: str) -> float | None:
+        """Get the target temperature of the climate device."""
+        response: dict[str, Any] = await self.aiolivisi.async_get_device_state(
```

Comment:
I think you fixed this wrong. The problem was in the type hint:
```suggestion
        response: dict[str, Any] | None = await self.aiolivisi.async_get_device_state(
```
I think you need to bring back the `None` check.

Reply:
I changed this, thank you.

Reply:
I changed this, thank you.